frontend/tests/page-object/login.page.ts

```diff
@@ -0,0 +1,11 @@
+import type { Page } from '../utils/helpers'
+
+export const LoginPage = {
+  email: (page: Page) => page.getByLabel('Email address'),
+  errorEmail: (page: Page) => page.locator('text=Please provide a valid email address.'),
+  errorInvalidPassword: (page: Page) => page.locator('text=Please enter your password.'),
+  errorPasswordMaxLength: (page: Page) => page.locator('text=Please provide at least 5 characters.'),
+  loginBtn: (page: Page) => page.locator('[aria-label="Log in"]'),
```

Comment:
this one is preferable:
```suggestion
  loginBtn: (page: Page) => page.getByRole('button', { name: 'Log in' }),
```